core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileArgs.scala

```diff
@@ -71,8 +71,9 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
   val platform: ScallopOption[String] =
     opt[String](required = false,
       descr = "Cluster platform where Spark GPU workloads were executed. Options include " +
-        s"${PlatformNames.getAllNames.mkString(", ")}. Default is ${PlatformNames.ONPREM}.",
-      default = Some(PlatformNames.ONPREM))
+        s"${PlatformNames.getAllNames.mkString(", ")}. " +
+        s"Default is ${PlatformFactory.getDefault.getName}.",
+      default = Some(PlatformFactory.getDefault.getName))
```

Comment:
nit: is this a good idea to create new instances of `OnPremPlatform` to get the names. This is just a thought.

Reply:
Updated the flow to use `PlatformNames.DEFAULT`.